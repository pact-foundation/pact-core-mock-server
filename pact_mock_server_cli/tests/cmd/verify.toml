bin.name = "pact-mock-server"
args = "verify --help"
