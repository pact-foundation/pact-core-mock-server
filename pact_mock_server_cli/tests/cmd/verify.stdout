Verify the mock server by id or port number, and generate a pact file if all ok

Usage: pact-mock-server verify [OPTIONS]

Options:
      --help
          Print help and exit
  -i, --mock-server-id <mock-server-id>
          the ID of the mock server
  -m, --mock-server-port <mock-server-port>
          the port number of the mock server
  -v, --version
          Print version information and exit
  -p, --port <port>
          port the master mock server runs on (defaults to 8080)
  -h, --host <host>
          hostname the master mock server runs on (defaults to localhost)
  -l, --loglevel <loglevel>
          Log level for mock servers to write to the log file (defaults to info) [possible values: error, warn, info, debug, trace, none]
      --no-term-log
          Turns off using terminal ANSI escape codes
      --no-file-log
          Do not log to an output file

